src/node_task_runner.cc

```diff
@@ -240,19 +219,66 @@ void ProcessRunner::Run() {
   uv_run(loop_, UV_RUN_DEFAULT);
 }
 
+std::optional<std::tuple<std::filesystem::path, std::string, std::string>>
+FindPackageJson(const std::filesystem::path& cwd) {
```

Comment:
Shouldn't it take into consideration the permission model when it's enabled? That's what happens on module loading for instance.

Reply:
We don't need to. `node --run` run outside of the permission model, and even node.js, because it gets executed as soon as `--run` is parsed as a CLI argument, and in the future, I'm thinking about stopping to initialize V8 and even OpenSSL.